project.properties

```diff
@@ -5,4 +5,5 @@
 #     export `export `cat ./aws-cryptographic-material-providers-library-dafny/project.properties`
 # The Java project include this file as a Gradle properties
 # And the Dotnet projects include and parse this file.
-dafnyVersion=4.2.0
\ No newline at end of file
+dafnyVersion=4.2.0
+dafnyVerifyVersion=4.2.0
```

Comment:
Question: Do we intend to bump this to `4.6.0`?

Reply:
We wanted to, but StandardLibrary/UTF8.ValidUTF8Concat doesn't verify in 4.6.0, and it's not clear what to do about it. It currently verifies under nightly, and so once 4.7 is available we'll try again.